examples/cnn/train.py

```diff
@@ -136,8 +136,14 @@ def run(global_rank,
         model = cnn.create_model(num_channels=num_channels,
                                  num_classes=num_classes)
     elif model == 'mlp':
-        from model import mlp
-        model = mlp.create_model(data_size=data_size, num_classes=num_classes)
+        import os, sys, inspect
```

Comment:
I have moved mlp into examples/mlp folder.
this folder is for cnn.
pls test the scripts in examples/mlp